api/v1/kogitoservices.go

```diff
@@ -334,11 +330,11 @@ func (k *KogitoServiceSpec) IsInsecureImageRegistry() bool { return k.InsecureIm
 func (k *KogitoServiceSpec) GetPropertiesConfigMap() string { return k.PropertiesConfigMap }
 
 // GetInfra ...
-func (k *KogitoServiceSpec) GetInfra() []string { return k.Infra }
+func (k *KogitoServiceSpec) GetInfra() []string { return nil }
 
 // AddInfra ...
 func (k *KogitoServiceSpec) AddInfra(name string) {
-	k.Infra = append(k.Infra, name)
+	panic("Operation not supported")
```

Comment:
Raising panic here will crash the operator pod.

Reply:
yes, I added them intentionally. These KogitoInfra related functions like `AddInfra` should not be called in any scenerio. If it is called then we need to fix that calling in code.

Reply:
Wouldn't it be better to log the error instead of panic?
This way any deployment in any namespace can bring the cluster wide operator down in production.

Reply:
I agree, even if user reach this point unintentionally it might crash operator leading to start the "restart loop" until user don't remove his CR, right?
maybe throw a error might be more sense